### PR TITLE
chore: reorganize compliance tests to always use same models

### PR DESCRIPTION
**Pull request by johannes-vogel**
@BobdenOs Do you see the problem with the hana/pg tests? I feel that views are not supported in CREATE.test.js? Should we simply skip to unblock us internally?

### CODE REVIEW

File: test/compliance/DELETE.test.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 const cds = require('../cds.js')
 
 describe('DELETE', () => {
-  cds.test(__dirname + '/../bookshop')
+  cds.test(__dirname + '/resources')
```
Comment (patricebender) [inline test/compliance/DELETE.test.js:6]: there is a similar test in `UPDATE.test.js`

File: test/compliance/resources/db/complex/index.cds [hunk 2]
```diff
@@ -11,3 +11,10 @@ entity Authors {
   name   : String(111);
   books  : Association to many Books on books.author = $self;
 }
+
```
Comment (patricebender) [inline test/compliance/resources/db/complex/index.cds:14]: we could clean up the `RenameKeys` projection in the bookshop then

File: test/compliance/DELETE.test.js (hunk 1 above)
Comment (johannes-vogel) [reply test/compliance/DELETE.test.js:6]: I don't see any :)

Comment (patricebender) [reply test/compliance/DELETE.test.js:6]: oops, thought https://github.com/cap-js/cds-dbs/pull/329/files was already merged

Comment (patricebender) [reply test/compliance/DELETE.test.js:6]: i will change it in the PR once this is in main :)

**Review by BobdenOs — APPROVED** [review]

**Pull request merged**